apps/web/src/components/docs/DocsButton.tsx

```diff
@@ -0,0 +1,111 @@
+import { ActionIcon, Popover, useMantineColorScheme } from '@mantine/core';
```

Comment:
Please avoid direct imports from `@mantine/core` as discussed [here](https://www.notion.so/novuhq/Using-PandaCSS-at-Novu-7b155517c2ff448ca65527fa7b2c9afe?pvs=4#fe38738ef93b42b5af98f66d0142b052).
```suggestion
import { useColorScheme } from '@novu/design-system';
```